.github/workflows/build-test.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       matrix:
         jdk: [ 11, 8 ]
-        scala: [ 2.12.15, 2.13.6, 3.0.0 ]
+        scala: [ 2.12.15, 2.13.7, 3.0.2 ] # Should be sync with Mergify conditions (.mergify.yml)
```

Comment:
Scala 2.13.8 out already (at least on Maven Central)

Reply:
I tried, but not all dependencies are ready for 2.13.8 🤷‍♂️

Reply:
:smile: Yes I see now in #694